rfc/ucan-delegation-handling.md

```diff
@@ -0,0 +1,167 @@
+# RFC: Standardizing UCAN Delegation Handling
+
+## Summary
+
+This RFC proposes standardizing how we handle UCAN delegations and proofs across our services, adopting CAR (Content-Addressed aRchive) files as the canonical format with base64 encoding for transport.
+
+## Problem Statement
+
+Our services currently accept UCAN delegations in multiple formats:
+
+- Base64 encoded strings.
+
+- Base64-encoded CIDv1 with embedded CAR data.
```

Comment:
This is the new format. In the CID, the multihash uses the identity codec and the [IPLD codec is CAR](https://github.com/multiformats/multicodec/blob/master/table.csv#L142).

This was an attempt to make the delegation self describing.

I appreciate this is somewhat onerous to create or parse.